### PR TITLE
g:buftabline_number = 2 to show sequential numbers

### PR DESCRIPTION
**Pull request by angn**
I wanted to make key bindings to jump to an other buffer quickly like below.
```
nn <leader>1 :exe 'b'.buftabline#user_buffers()[0]<cr>
nn <leader>2 :exe 'b'.buftabline#user_buffers()[1]<cr>
nn <leader>3 :exe 'b'.buftabline#user_buffers()[2]<cr>
```

However, I found it's hard to know the sequential index of a buffer when there are a lot of ones, so I added some code to show sequential numbers.

### CODE REVIEW

**Comment by ap** [comment]
Thanks for the patch. I plan to take it but maybe not verbatim. Something about it bugs me; unfortunately I can’t put my finger on it yet. (Also it’ll need docs.)

(I’m not sure I like the interface of setting `g:buftabline_number` to a special value, though it would be consistent with the other options… which I don’t like either but there too I mimicked the interface of Vim’s built-in stuff for consistency. Ugh?)

Basically I’ve got a bunch of obnoxious æsthetic quibbles with it even though there’s nothing objectively wrong with it; allow me to beg indulgence for my irrationalities.

Anyway. The feature will go in soon in whichever form. Again, thanks for the patch.

**Comment by ap** [comment]
Sorry this almost fell through the cracks. But it’s now done.
